src/openzaak/components/autorisaties/migrations/0011_autorisatiespec_to_catalogusautorisatie.py

```diff
@@ -0,0 +1,76 @@
+# SPDX-License-Identifier: EUPL-1.2
+# Copyright (C) 2024 Dimpact
+# Generated by Django 4.2.11 on 2024-07-15 10:00
+from django.db import migrations
+
+
+def migrate_autorisatiespecs_to_catalogus_autorisaties(apps, schema_editor):
```

Comment:
I'm not sure if I missed anything in this migration, so if you can think of anything that might be missing, please mention it because this will be a one time run and we don't want to run into issues with it :grimacing: